packages/cli/src/spec/configSpec.ts

```diff
@@ -30,14 +30,14 @@ const LEGACY_CONFIG_FOLDER = join(homedir(), '.llama-pack');
 const LEGACY_CONFIG_NAME = 'llama-pack-config.json';
 const LEGACY_CONFIG_FILE_PATH = join(LEGACY_CONFIG_FOLDER, LEGACY_CONFIG_NAME);
 
-beforeAll(() => {
-  spyOn(JdkHelper, 'validatePath').and.returnValue(Promise.resolve(Result.ok('/path/to/jdk')));
-  spyOn(AndroidSdkTools, 'validatePath').and.returnValue(Promise.resolve(Result.ok(
-      '/path/to/android-sdk')));
-});
-
 describe('config', () => {
   describe('#loadOrCreateConfig', () => {
+    beforeAll(() => {
+      spyOn(JdkHelper, 'validatePath').and.returnValue(Promise.resolve(Result.ok('/path/to/jdk')));
+      spyOn(AndroidSdkTools, 'validatePath').and.returnValue(Promise.resolve(Result.ok(
```

Comment:
Do you mind line breaking this at `.returnValue` - I know you didn't write this originally but I think it reads a bit better:

```
spyOn(AndroidSdkTools, 'validatePath').and
    .returnValue(Promise.resolve(Result.ok('/path/to/android-sdk')));
```

Reply:
Related and worth reading: https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break